proto/connectrpc/conformance/v1alpha1/service.proto

```diff
@@ -38,15 +45,21 @@ service ConformanceService {
   //
   // If the input stream is empty, the server's response will include no data,
   // only the request properties (headers, timeout).
-  rpc ClientStream(stream ClientStreamRequest) returns (ConformancePayload);
+  //
+  // Servers should only read the response definition from the first message in
+  // the stream and should ignore any definition set in subsequent messages.
```

Comment:
Should we allow clients to specify this in the subsequent messages and just ignore it? Or should we mandate that a server throws an error if anything is specified in subsequent messages?

Reply:
I think it's fine if the server just ignores the data. In practice, the actual test cases will never have requests that have that, so it's moot how the server behaves. If we want to trigger an "invalid argument" error in the server, that will be done from the response definition, not from sending a bad request.

Reply:
Makes sense. I'll leave the comments as-is.